releasenotes/notes/sbplx_optimizer-a27361fc4ac8de8c.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support for SBPLX optimizer from NLopt library has been added.
+    SBPLX is a local gradient-free optimizer based on Nelder-Mead and
+    is expected to show better convergence behavior.
+    Further information about this and all other optimizers can be found
+    `in the API ref. here <https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html>__`.
```

Comment:
```suggestion
    `in the API ref. here <https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html>`__.
```
This is not coming out right in the docs. The trailing tick is in the wrong place it seems.

![image](https://github.com/user-attachments/assets/533d4fdd-056c-44f2-a18e-02561536a7bc)